decidim-comments/config/locales/de-DE.yml

```
@@ -75,23 +75,23 @@ de:
         loading: Loading comments ...
         title: "%{count} comments"
       featured_comment:
-        title: Featured comment
+        title: Wichtigster Kommentar
     events:
       comments:
         comment_created:
           comment:
-            email_intro: '"%{resource_title}" has been commented. You can read the comment in this page:'
+            email_intro: '"{resource_title}" wurde kommentiert. Sie können den Kommentar auf dieser Seite lesen'
```

Hi, thank you for your contribution

Is the deletion of the "%" character voluntary ?

```suggestion
            email_intro: '"%{resource_title}" wurde kommentiert. Sie können den Kommentar auf dieser Seite lesen'
```